modules/m_info.c

```diff
@@ -200,6 +200,11 @@ static struct InfoStruct info_table[] = {
 		"Message to quit users with if this server is draining.",
 		INFO_STRING(&ConfigFileEntry.drain_reason),
 	},
+        {
+                "sasl_only_client_message",
+                "Message to quit users with if they require SASL authentication.",
+                INFO_STRING(&ConfigFileEntry.sasl_only_client_message),
+        },
```

Comment:
Same here.

Reply:
Serves me right for using nano instead of an actual editor i guess.

Reply:
Hey! I've edited `doc/reference.conf` with nano several times. :P